File: src/main/java/net/lomeli/lomlib/core/command/CommandClearHostiles.java
package net.lomeli.lomlib.core.command;

import java.util.List;

import net.minecraft.command.CommandException;
import net.minecraft.command.ICommandSender;
import net.minecraft.entity.Entity;
import net.minecraft.entity.monster.IMob;

public class CommandClearHostiles extends CommandBaseLomLib {
    @Override
    public String getCommandName() {
        return "clear-hostiles";
    }

    @Override
    public int getRequiredPermissionLevel() {
        return 2;
    }

    @Override
    public void processCommand(ICommandSender sender, String[] args) throws CommandException {
        if (sender == null) return;
        List<?> entityList = sender.getEntityWorld().loadedEntityList;
        if (entityList == null || entityList.isEmpty()) return;
        for (Object entity : entityList) {
            if (entity instanceof Entity) {
                if (entity instanceof IMob)
                    ((Entity) entity).setDead();
            }
        }
        sendMessage(sender, "command.lomlib.clear-hostiles.success");
    }
}

File: src/main/kotlin/net/lomeli/lomlib/core/command/CommandClearHostiles.kt
package net.lomeli.lomlib.core.command

import net.minecraft.command.CommandException
import net.minecraft.command.ICommandSender
import net.minecraft.entity.Entity
import net.minecraft.entity.monster.IMob
import net.minecraft.server.MinecraftServer

class CommandClearHostiles : CommandBaseLomLib() {
    override fun getName(): String = "clear-hostiles"

    override fun getRequiredPermissionLevel(): Int = 2

    @Throws(CommandException::class)
    override fun execute(server: MinecraftServer, sender: ICommandSender, args: Array<String>) {
        if (sender == null) return
        val entityList = sender.entityWorld.loadedEntityList
        if (entityList == null || entityList.isEmpty()) return
        for (entity in entityList) {
            if (entity is Entity) {
                if (entity is IMob)
                    entity.setDead()
            }
        }
        sendMessage(sender, "command.lomlib.clear-hostiles.success")
    }
}